### ISSUE TITLE
app-crypt/elfcrypt-20200224 does not respect CFLAGS

### ISSUE THREAD

**Issue by asarubbo** [pentoo/pentoo-overlay#762]
https://blogs.gentoo.org/ago/2020/07/04/gentoo-tinderbox/

Issue: app-crypt/elfcrypt-20200224 does not respect CFLAGS.
Discovered on: amd64 (internal ref: pentoo_tinderbox)

NOTE:
To reproduce this issue you may want to add "-frecord-gcc-switches" to CFLAGS/CXXFLAGS/FFLAGS/FCFLAGS. If this is not something about c/c++ context see this bug as an hint to hide the QA warning (with QA_FLAGS_IGNORED) where is not possible to respect CFLAGS.

At the time of writing, github-cli does not support attachments. I'm using gists but they can be deleted at a certain point. If you want to save them, please create attachments on your own.

build log: https://gist.github.com/14f484be4af431b0a6dd341719e1089d

**Comment by blshkv** [comment]
Yes, yes, we know. Our ebuilds are not perfect, that's why it is still not in Gentoo. Please create a pull request with a fix you care

**Comment by ZeroChaos-** [comment]
Please do not close valid bug reports without fixing.  I care, which is why I asked for this tinderbox run
